sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntity.cs

```diff
@@ -68,8 +64,8 @@ internal HealthcareEntity(HealthcareEntityInternal entity, IReadOnlyDictionary<H
         public IReadOnlyCollection<EntityDataSource> DataSources { get; }
 
         /// <summary>
-        /// Gets the dictionary for related entity with mapped relation type for each.
+        /// Gets the entities and the relationship between the entities.
         /// </summary>
-        public IReadOnlyDictionary<HealthcareEntity, HealthcareEntityRelationType> RelatedEntities { get; }
+        public Dictionary<HealthcareEntity, HealthcareEntityRelationType> RelatedEntities { get; }
```

Comment:
could u revert this to `IReadOnlyDictionary` ? public types we expose tu customers are readonly

Reply:
@maririos I tried, but I am getting an error when I add key value in transform class

```
                sourceEntity.RelatedEntities.Add(targetEntity, relationType);
```